.drone.star

```diff
@@ -14,65 +14,74 @@ windowsglobalimage="cppalliance/dronevs2019"
 
 def main(ctx):
   return [
+  # Coverage
```

Comment:
Please paste the chart we made, at the top of this function
```
  # Priorities: (no 2a, no betas)
  #
  # coverage
  # latest gcc: 11,14,17,20
  # latest clang: 11,14,17,20
  # oldest gcc: 4.8
  # oldest clang: 3.8
  # valgrind
  # asan
  # tsan
  # ubsan
  # docs
  # arm64
  # s390x
  # (...the rest)
```

Reply:
I interleaved it with the jobs. But I'll put it there.